app/vue/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@storybook/vue",
+  "version": "3.0.0",
+  "description": "Storybook for Vue: Develop Vue Component in isolation with Hot Reloading.",
+  "homepage": "https://github.com/storybooks/storybook/tree/master/apps/vue",
+  "bugs": {
+    "url": "https://github.com/storybooks/storybook/issues"
+  },
+  "license": "MIT",
+  "main": "dist/client/index.js",
+  "bin": {
+    "build-storybook": "./dist/server/build.js",
+    "start-storybook": "./dist/server/index.js",
+    "storybook-server": "./dist/server/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybooks/storybook.git"
+  },
+  "scripts": {
+    "dev": "DEV_BUILD=1 nodemon --watch ./src --exec 'npm run prepublish'",
+    "prepublish": "node ../../scripts/prepublish.js"
+  },
+  "dependencies": {
+    "@storybook/addon-actions": "^3.1.2",
+    "@storybook/addon-links": "^3.1.2",
+    "@storybook/addons": "^3.1.2",
+    "@storybook/channel-postmessage": "^3.1.2",
+    "@storybook/ui": "^3.1.3",
+    "airbnb-js-shims": "^1.1.1",
+    "autoprefixer": "^7.1.1",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.0.0",
+    "babel-plugin-react-docgen": "^1.5.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
It's unnecessary to have es2015, es2016 here and in .babelrc (I noticed this for all the apps (react/react-native) - can switch to preset-env instead.

`"es2015", "es2016"` -> `"env"`